pkg/operation/botanist/logging.go

```diff
@@ -93,9 +92,29 @@ func (b *Botanist) DeploySeedLogging(ctx context.Context) error {
 
 	sgFluentdPasswordHash = string(sgFluentdSecret.Data["bcryptPasswordHash"])
 
+	var (
+		defaultSecretName  = "kibana-tls"
```

Comment:
Maybe expose this as a constant somewhere and reuse it here and in `pkg/operation/botanist/secrets.go` where it is generated?

Reply:
I moved it over to `pkg/operation/common/types.go` because I ran into cyclic dependency issues with `pkg/operation/botanist/secrets.go`.